mlflow/sklearn/__init__.py

```diff
@@ -368,6 +372,7 @@ def log_model(
                             waits for five minutes. Specify 0 or None to skip waiting.
     :param pip_requirements: {{ pip_requirements }}
     :param extra_pip_requirements: {{ extra_pip_requirements }}
+    :param pyfunc_predict_func: the prediction function to use for the pyfunc flavor
```

Comment:
Nit: can we make this docstring consistent with the one in `mlflow.sklearn.save_model()`?